test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -49,3 +62,57 @@ async def test_topology_ops(request, manager: ManagerClient):
     logger.info(f"Decommissioning node {servers[0]}")
     await manager.decommission_node(servers[0].server_id)
     await check_token_ring_and_group0_consistency(manager)
+
+    logger.info("Checking results of the background writes")
+    await finish_writes()
+
+    for server in servers:
+        await check_node_log_for_failed_mutations(manager, server)
+
+
+async def check_node_log_for_failed_mutations(manager: ManagerClient, server: ServerInfo):
+    logger.info(f"Checking that node {server} had no failed mutations")
+    log = await manager.server_open_log(server.server_id)
+    occurrences = await log.grep(expr="Failed to apply mutation from", \
+                                 filter_expr="replica::stale_topology_exception") # Disabled due to #15804
```

Comment:
Please remember to refer to this place in the code from #15804 once the PR is merged (so whoever fixes #15804 will remember to update the test)